three.c

```diff
@@ -0,0 +1,56 @@
+/*Write a program three.c that takes in 3 user-input integers and checks whether or not the 3 integers form a valid triangle (no one side is longer than the sum of the other 2 sides). It should contain the following functions:
+
```

Comment:
Great job! Personally would have handled floats differently (not truncated), but awesome job! 🎉